src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs

```diff
@@ -126,6 +128,23 @@ protected override string GenerateCommandLineCommands ()
 		{
 			var cmd = GetCommandLineBuilder ();
 
+			if (NamespaceTransforms?.Any () == true) {
+				// ex: obj/Debug/generated/msbuild-metadata.xml
+				var transform_file = Path.Combine (OutputDirectory, "..", "msbuild-metadata.xml");
```

Comment:
Does this file need to be added to `FileWrites` ? It will be deleted on the next build otherwise.

Reply:
The entire generated output directory is deleted on every build:

https://github.com/xamarin/xamarin-android/blob/main/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets#L53-L54